app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/room/db/constant/DatabaseContract.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2017 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package android.support.v17.leanback.supportleanbackshowcase.app.room.db.constant;
+
+
+public class DatabaseContract {
```

Comment:
why is this necsssary? do you have a content provider?

Reply:
Oh my bad. This class has been renamed as DatabaseColumnConstant for clarification.

Reply:
Oh, my bad. This class has been renamed to DatabaseColumnConstant for clarification